src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurExpenseV3ServiceImpl.java

```diff
@@ -167,9 +165,22 @@ protected void updateStatusInConcur(String accessToken, String reportId, boolean
         String logMessageDetail = buildLogMessageDetailForExpenseWorkflowAction(workflowAction, reportId);
         ConcurWebRequest<Void> webRequest = buildWebRequestForExpenseWorkflowAction(
                 workflowAction, reportId, resultsDTO);
+        
+        /*
+         * @todo re enable this
+         */
```

Comment:
this is the code I mentioned in the PR description about local debug code that needs to be removed before the PR can be merged.
it can be removed now